test/test.wpcom.js

```diff
@@ -20,4 +20,75 @@ describe( 'wpcom', function() {
 				.catch( done );
 		} );
 	} );
+
+	describe( 'wpcom.plans', function() {
+		describe( 'wpcom.plansList', function() {
+			it( '[1] should get available WordPress.com plans', function( done ) {
+				var wpcom = util.wpcomPublic();
+
+				wpcom.plansList( { apiVersion: '1' } )
+					.then( data => {
+						// testing object
+						assert.ok( data );
+						assert.ok( data instanceof Array );
+						done();
+					} )
+					.catch( done );
+			} );
+
+			it( '[1.2] should get available WordPress.com plans', function( done ) {
+				var wpcom = util.wpcomPublic();
+
+				wpcom.plansList( { apiVersion: '1.2' } )
+					.then( data => {
+						// testing object
+						assert.ok( data );
+						assert.ok( data instanceof Array );
+						done();
+					} )
+					.catch( done );
+			} );
+
+			it( '[1.3] should get available WordPress.com plans', function( done ) {
+				var wpcom = util.wpcomPublic();
+
+				wpcom.plansList( { apiVersion: '1.3' } )
```

Comment:
I'm not sure why and what are the differences between these three versions of the same endpoint. The fact is that they are defined in the WP REST-API.
I think we should try to keep in mind these versions and maybe in some moment start to set the api version for each test since the response could be slightly different.